src/alchemlyb/parsing/gmx.py

```diff
@@ -105,6 +105,9 @@ def extract_u_nk(xvg, T):
 
     u_k.name = 'u_nk'
 
+    u_k.attrs['temperature'] = T
+    u_k.attrs['energy_unit'] = 'kT'
```

Comment:
Does this work without a warning with pandas ≥ 1.0?

Reply:
It works without any warning.